### PR TITLE
allow release managers + infra/release/testing leads to trigger image…

### PR DESCRIPTION
**Pull request by BenTheElder**
… promo re-runs

we can reasonably trust these folks to not do this pointlessly (in no particular order)

### CODE REVIEW

File: config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml [hunk 1]
```diff
@@ -26,6 +26,16 @@ postsubmits:
       testgrid-alert-email: k8s-infra-alerts@kubernetes.io, release-managers+alerts@kubernetes.io
       testgrid-num-failures-to-alert: '2'
   - name: post-k8sio-image-promo
+    rerun_auth_config:
+      github_team_slugs:
+      - org: kubernetes
+        slug: sig-k8s-infra-leads
+      - org: kubernetes
+        slug: sig-k8s-testing-leads
```
Comment (justaugustus) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml:34]:
```suggestion
        slug: sig-testing-leads
```

File: config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml [hunk 2]
```diff
@@ -26,6 +26,16 @@ postsubmits:
...
+      - org: kubernetes
+        slug: sig-k8s-release-leads
```
Comment (justaugustus) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml:36]:
```suggestion
        slug: sig-release-leads
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
quick nits + lexicographical sort?

**Comment by BenTheElder** [comment]
/hold

shrug on sort, I don't think we're likely to attempt auto-formatting this and absent auto-formatting manually sorting new entries is tedious, but definitely a typo on the one team entry ... fixing

File: config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml (hunk 2 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml]: d'oh :(

fixed this and above.

**Review by justaugustus — APPROVED** [review]
/lgtm
/approve
FYI @kubernetes/sig-release-leads @kubernetes/release-engineering

**Comment by BenTheElder** [comment]
re-posting from slack:

> There's a little :leftwards_arrow_with_hook: type button at the top of the job results page that will let you re-run a job with github OAUTH if you're in these teams.
>
> You can select between the config the job ran with, or the latest config for the job (e.g. if we patched the config and want to start a new run immediately with the fixed job config)
>
> Useful for this sort of scenario [unsticking a failed image promotion due to quota exhaustion with a retry]

cc also @kubernetes/release-managers @kubernetes/sig-testing-leads @kubernetes/sig-k8s-infra-leads for visibility

**Pull request merged**